site/index.html

```diff
@@ -0,0 +1,95 @@
+      <img src="images/logo.png" alt="My website logo">
+    </a>
+    <h3>My Favourite Games</h3>
+    <ol>
+      <li>Cricket</li>
+      <li>Valley Ball</li>
+      <li>Badminton</li>
+    </ol>
+    <section id="haiku">
+      <h3>Haiku</h3>
+      <p>
+        Haikus are easy,<br>
+        But somtimes they do not make sense,<br>
+        Refrigerator.<br>
+        <cite>by <span class="author">Rolph Nelson</span></cite>
+      </p>
+    </section>
+    <section id="embedding-external-content">
+      <h3>Public Domain Audiobook</h3>
+      <audio controls>
+        <source src="https://www.archive.org/download/money_and_trade_considered_2111_librivox/moneyandtrade_01_law_128kb.mp3" type="audio/mpeg">
+        <a href="https://www.archive.org/download/money_and_trade_considered_2111_librivox/moneyandtrade_01_law_128kb.mp3" download>Download the audiobook</a>.
+      </audio>
+      <h3>YouTube Video</h3>
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/Uw_QyeHo8f0?si=dYoTBJf9Rf58QlrH" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+    </section>
+  </main>
+  <footer>
+    <hr>
+    <blockquote>
+      "Tell me what you eat and I will tell you what you are."&mdash;<cite><span class="author">Anthelme Brillat-Savarin</span></cite>
+    </blockquote>
+    <p>&copy; 2025 <a href="mailto:24044778@saultcollege.ca?subject=Message from CSD112 Kishan's Website">Kishankumar Patel</a></p>
+    <section id="Working-with-Image-Support ">
+      <picture>
+        <source srcset="images/sc-logo/sc-logo.svg" type="image/svg+xml">
+        <image alt="sc-logo"
+               source="images/sc-logo/sc-logo.png"
+               srcset="images/sc-logo/sc-logo.png 1x,images/sc-logo/sc-logo@1.5x.png 1.5x,images/sc-logo/sc-logo@2x.png 2x"
+               width="70px">
```

Comment:
> 11. `Lab 3 > Working with Image Support > picture element > has a fallback img with > sc-logo as its src`
```suggestion
        <img alt="sc-logo"
               source="images/sc-logo/sc-logo.png"
               srcset="images/sc-logo/sc-logo.png 1x,images/sc-logo/sc-logo@1.5x.png 1.5x,images/sc-logo/sc-logo@2x.png 2x">
```
Wrong tag is used should be img instead